Definitions/BlueMage.json

```diff
@@ -93,16 +98,22 @@
 ] },
 { "6B9" : "peculiar light", "potency": [ {"type": "DamageReceivedMultiplier", "amount": 5, "limittodamagetype": "Magic"} ] },
 { "6BA" : "diamondback", "potency": [ {"type": "DamageReceivedMultiplier", "amount": -90} ] },
-{ "846" : "harmonized", "potency": [ {"type": "damagedonemultiplier", "amount": 80, "limittodamagetype": "Physical"} ] },
-{ "847" : "cactguard", "potency": [ {"type": "DamageReceivedMultiplier", "amount": 5} ] },
-{ "849" : "physical attenuation", "potency": [ {"type": "DamageReceivedMultiplier", "amount": 5} ] },
+{ "846" : "harmonized", "potency": [ {"type": "damagedonemultiplier", "amount": 80, "limittodamagetype": "Physical", "limittoactioncategory": "spell"} ] },
+{ "847" : "cactguard", "potency": [ {"type": "DamageReceivedMultiplier", "amount": -5} ] },
+{ "84B" : "physical attenuation", "potency": [ {"type": "DamageReceivedMultiplier", "amount": 5, "limittodamagetype": "Physical"} ] },
 { "84E" : "aetherial mimicry: healer", "potency": [ {"type": "HealDoneMultiplier", "amount": 20} ] },
 { "9BC" : "tingling", "potency": [ {"type": "DamageAddPotency", "amount": 100, "limittodamagetype": "Physical"} ] },
 
-{ "6B8": "ice spikes", "reactiveproc": { "type": "DamageOnDamageReceived" } },
-{ "6BC": "veil of the whorl", "reactiveproc": { "type": "DamageOnDamageReceived" } },
+{ "6B8": "ice spikes", "reactiveproc": { "type": "DamageOnDamageReceived", "amount": 40 } },
+{ "6BC": "veil of the whorl", "reactiveproc": { "type": "DamageOnDamageReceived", "amount": 50 } },
 
 { "852" : "Surpanakha's Fury", "potency": [ {"type": "damagedonemultiplier", "amount": 50, "isstacked": true, "limitto": "Surpanakha"} ] },
+    
+{ "6B3" : "Bad Breath", "potency": [ {"type": "damagedonemultiplier", "amount": -10} ] },
+{ "12"  : "Bad Breath", "timeproc": { "type": "dot", "potency": "20", "damagetype": "magic", "maxticks": "5" } },
+{ "842" : "Gobskin", "damageshield": { "type": "Potency", "amount": 0 } },
+{ "843" : "Magic Hammer", "potency": [ { "type": "damagedonemultiplier", "amount": -10, "limittodamagetype": "magic"} ] },

```

Comment:
Magic Hammer gives the 843 Conked debuff which reduces intelligence and mind by 10%